match/spec/portal_fetcher_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spaceship/client'
+
+describe Match do
+  describe Match::Portal::Fetcher do
+    let(:default_sut) {
+      Match::Portal::Fetcher
+    }
+
+    let(:portal_bundle_id) { double("portal_bundle_id") }
+    let(:portal_device) { double("portal_device") }
+    let(:portal_certificate) { double("portal_certificate") }
+
+    let(:portal_profile) { double("portal_profile") }
+    let(:portal_profile_udid) { double("portal_profile_udid") }
+
+    let(:deviceClass) { Spaceship::ConnectAPI::Device::DeviceClass }
+    let(:deviceStatus) { Spaceship::ConnectAPI::Device::Status }
+    let(:certificateType) { Spaceship::ConnectAPI::Certificate::CertificateType }
+    let(:profileState) { Spaceship::ConnectAPI::Profile::ProfileState }
+
+    let(:default_device_all_params) {
+      {
+        filter: { platform: 'IOS,UNIVERSAL', status: 'ENABLED' },
+        client: anything
+      }
+    }
+
+    let(:default_certificates_all_params) {
```

Comment:
Nit: if I'm not mistaken it's not standard to use the `{ … }` syntax for multiline blocks in Ruby, see: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockDelimiters

We punted the enabling of this rule in fastlane's rubocop just to avoid fixing the (then 16) use cases of such bad usage at the time rubocop was implemented, but I think we should stick to this good practice 🙏

Reply:
Yes, I always forget about this when switching from Swift 😅. I will change it tomorrow to `do` and `end`.

Reply:
Done in 264a56b7e2e7012ae2830852482c9f6a2bc6487e.